src/utils/auth.ts

```diff
@@ -28,13 +28,20 @@ const getDebugUser = async (cookies: NextApiRequest['cookies']): Promise<User |
     return cookie ? userDbService.getByEmail(cookie) : undefined;
 };
 
-const getDebugRoles = (cookies: NextApiRequest['cookies']): Promise<UserRolesInfo> | undefined => {
+const getDebugRoles = async (cookies: NextApiRequest['cookies']): Promise<UserRolesInfo | undefined> => {
     const cookie =
         standConfig.isDebugCookieAllowed && ROLE_DEBUG_COOKIE_NAME in cookies
             ? cookies[ROLE_DEBUG_COOKIE_NAME]
             : undefined;
 
-    return cookie ? externalUsersDebugService.getUserRolesFromDebugCookie(cookie) : undefined;
+    const user = await getDebugUser(cookies);
+    const debugRoles = cookie ? externalUsersDebugService.getUserRolesFromDebugCookie(cookie) : undefined;
```

Comment:
why is it here?

Reply:
you right, I'll make another issue an pr for that